datamade/static-app-template
# Initialize app template

## Overview

Initialize a static app template with a basic Gatsby installation, including:

- Containerized Node.js environment
- Bootstrap
- Fontawesome
- Helmet
- Basic Gatsby plugins:
    - Images
    - Offline
    - Filesystem

## Testing instructions

- Confirm that linters pass in Travis
- Confirm that the Netlify deploy preview builds successfully
